lib/label.ml

```diff
@@ -43,33 +43,36 @@ module Relation = struct
     | "<=" -> Le
     | "<" -> Lt
     | "=" -> Eq
-    | _ -> assert false (* can not happen, filtered by the regexp *)
+    | _ -> assert false
```

Comment:
a bit unfortunate but it should do the right thing if you put the comment before assert

Reply:
Good catch, I didn't really review the whole diff, I'll fix that!